src/style.css

```diff
@@ -1,3 +1,72 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, serif;
```

Comment:
Consider adding a fallback font for 'Roboto' to ensure text is displayed correctly if 'Roboto' is not available. For example: `font-family: Roboto, Arial, sans-serif;`